docs/sphinx/source/whatsnew/v0.9.5.rst

```diff
@@ -95,3 +89,5 @@ Contributors
 * Areeba Turabi (:ghuser:`aturabi`)
 * Mark Campanelli (:ghuser:`markcampanelli`)
 * Taos Transue (:ghuser:`reepoi`)
+* Tim Townsend
```

Comment:
@cwhanse I wonder if you have an opinion regarding this addition

Reply:
It seems appropriate to me